components/ide/jetbrains/image/gha-update-image/update-versions.ts

```diff
@@ -0,0 +1,105 @@
+import { $ } from "bun";
+import yaml from "yaml";
+import { z } from "zod";
+
+$.nothrow(); // git likes to respond with non-zero codes, but it is alright for us
+
+const versionManifest = z.object({
+    components: z.object({
+        workspace: z.object({
+            desktopIdeImages: z.record(z.string(), z.object({ version: z.string() })),
+        }),
+    }),
+});
+
+const workspaceYaml = z.object({
+    defaultArgs: z.object({
+        intellijDownloadUrl: z.string(),
+        golandDownloadUrl: z.string(),
+        pycharmDownloadUrl: z.string(),
+        phpstormDownloadUrl: z.string(),
+        rubymineDownloadUrl: z.string(),
+        webstormDownloadUrl: z.string(),
+        riderDownloadUrl: z.string(),
+        clionDownloadUrl: z.string(),
+    }),
+});
+
+const tagInfo =
+    await $`git ls-remote --tags --sort=-v:refname https://github.com/gitpod-io/gitpod | grep 'main-gha.' | head -n1`.text();
+const installationVersion =
+    await $`echo ${tagInfo} | awk '{ print $2 }' | grep -o 'main-gha.[0-9]*' | cut -d'/' -f3`.text();
+const versionData =
+    await $`docker run --rm eu.gcr.io/gitpod-core-dev/build/versions:${installationVersion.trim()} cat /versions.yaml`.text();
+
+const parsed = versionManifest.parse(yaml.parse(versionData));
+
+const pathToConfigmap = "../../../../../install/installer/pkg/components/ide-service/ide-configmap.json"; // this is insane
```

Comment:
should not we resolve relative to __dirname, that it works does not matter what is cwd?